.github/workflows/staging-pipeline.yml

```
@@ -117,36 +117,145 @@
+          'adapters',
+          'resolvers',
+          'services',
+          'entities',
+          'server',
+          'utils',
+          'migrations'
+        ]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v1
```

actionlint (1.7.7)

the runner of "actions/setup-node@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)